src/Microsoft.DotNet.Cli.Utils/CommandResolution/OutputPathCommandResolver.cs

```diff
@@ -80,7 +80,7 @@ private ProjectContext GetProjectContextFromDirectory(string directory, NuGetFra
             var projectContext = ProjectContext.Create(
                 projectRootPath,
                 framework,
-                DotnetRuntimeIdentifiers.InferCurrentRuntimeIdentifiers());
+                RuntimeEnvironmentRidExtensions.GetAllCandidateRuntimeIdentifiers());
```

Comment:
Why is it that this does not need the RID cap from InferCurrentRuntimeIdentifiers?